frontend/public/locales/en/translation.json

```diff
@@ -375,5 +375,49 @@
     "videos": "videos",
     "entityCandidate": "candidate",
     "entityVideo": "video"
+  },
+  "presidentielle2022": {
+    "dialogs": {
+      "tutorial": {
+        "title1": "Welcome on Tournesol \uD83C\uDF3B",
+        "message1": {
+          "p10": "In order to familiarize yourself with the comparison system, small messages like this one will be displayed.",
+          "p20": "Let's start with a first comparison between two candidates.",
+          "p30": "Move the blue handle below the candidates, then submit to select the person who should be president.",
```

Comment:
The French version specifies "d'après vous". I would prefer to also add here "who should be presnident according to you".